code/game/gamemodes/miniantags/space_turkey/space_turkey_abilities.dm

```diff
@@ -0,0 +1,103 @@
+#define MAX_TURKEY_TACKLE_DIST 6
+
+/obj/effect/proc_holder/spell/targeted/turkey_tackle
```

Comment:
Hmm, do you think it's better like this, or as a subtype of the existing leap spell in goon_powers.dm, perhaps modifying behaviors via overriding certain variables and handlers?

Reply:
I thought about that, but I couldn't think of a good way to append the whole "crashing into someone" bit. Not to mention the turkey's leap temporarily turns on MOBPASS, something the goon's Leap doesn't do.

Reply:
You could add a `on_collision` method to the leap, as well as a var that when set toggles MOBPASS

Reply:
Perhaps, but unless there's a good advantage to making it a subtype of leap, I'll probably leave it as-is. I know how this one works, and I'm reluctant to break down, remake, and retest the whole thing.